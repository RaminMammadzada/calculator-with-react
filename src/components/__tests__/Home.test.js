import ReactTestRenderer from 'react-test-renderer';
import Home from '../Home';

it('Creates a snapshot for Home Page and it should be the same with the previous snapshot', () => {
  const tree = ReactTestRenderer.create(<Home />).toJSON();
  expect(tree).toMatchSnapshot();
});
