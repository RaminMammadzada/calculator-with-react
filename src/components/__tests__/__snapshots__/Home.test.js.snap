// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Creates a snapshot for Home Page and it should be the same with the previous snapshot 1`] = `
<div>
  <h2>
    Math Magitian Calculator Homepage
  </h2>
  <p>
    Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
  </p>
</div>
`;
